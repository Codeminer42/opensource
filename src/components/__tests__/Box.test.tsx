import React from 'react'
import renderer from 'react-test-renderer'
import Box from '../Box'

describe('Box', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Box backgroundColor='black' />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
