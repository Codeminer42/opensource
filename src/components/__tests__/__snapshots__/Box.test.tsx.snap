// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box matches snapshot 1`] = `
<div
  className="Box-sc-qv0gud-0 gBFWCr"
/>
`;
